optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -41,10 +43,6 @@ As you can see, it is not optimal, as it does not leave enough room to assign th
 Planner does find the more optimal solution by using additional, smarter algorithms.
 It also scales: both in data (more processes, more computers) and constraints (more hardware requirements, other constraints). So let's see how Planner can be used in this scenario.
 
-
-[[cloudBalancingValueProposition]]
-=== Value Proposition
```

Comment:
The examples in chapter 3 each have this value proposition section, so it shows up in the TOC. If it's removed here, it needs to be removed there too.